index.bs

```diff
@@ -76,773 +93,795 @@ All <{iframe}> use cases are still valid and should continue to use <{iframe}>,
-## Clearing user content after each session ## {#clearing-session-data}
-<!-- ====================================================================== -->
+    // Navigation methods.
+    Promise<undefined> back();
+    boolean canGoBack();
+    boolean canGoForward();
+    Promise<undefined> forward();
+    Promise<undefined> go(long relativeIndex);
+    undefined reload();
+    undefined stop();
 
-In some environments, someone only uses a single device for a brief time to
-complete their task, like ordering in a restaurant. When their task is complete,
-the embedder application should be able to clear all of the local user data
-associated with the task and then restart the embedded instance.
+    // Scripting methods.
+    undefined addContentScripts(sequence<ContentScriptDetails> contentScriptList);
+    Promise<any> executeScript(optional InjectDetails details = {});
+    Promise<undefined> insertCSS(optional InjectDetails details = {});
+    undefined removeContentScripts(sequence<DOMString>? scriptNameList);
 
-<!-- ====================================================================== -->
-## Monitor for idle sessions ## {#monitor-idle-sessions}
-<!-- ====================================================================== -->
+    // Configuration methods.
+    Promise<undefined> clearData(
+      optional ClearDataOptions options = {},
+      optional ClearDataTypeSet types = {});
+    Promise<boolean> getAudioState();
+    Promise<long> getZoom();
+    Promise<boolean> isAudioMuted();
+    undefined setAudioMuted(boolean mute);
+    Promise<undefined> setZoom(long zoomFactor);
 
-While users interact with embedded content, the user may not explicitly end
-their session. This content may assume the user is present when they have
-actually finished or departed without completing the task. Embedder applications
-want to detect when users idle over their case's threshold and begin a fresh
-session.
+    // Capture methods.
+    undefined captureVisibleRegion();
+    undefined print();
+};
+</xmp>
+</dd>
+</dl>
 
-<!-- ====================================================================== -->
-## Arbitrarily blocking navigations ## {#blocking-navigations}
-<!-- ====================================================================== -->
+The <{controlledframe}> element [=represents=] its [=fenced navigable container/fenced navigable=].
 
-While displaying embedded web content that's not authored by the embedder, pages
-may link to third party web content that's disallowed. Allowing the embedder to
-edit elements in embedded content through arbitrary script injection into the
-web content can ensure navigation cannot occur to blocked pages. The embedder
-can also use the Controlled Frame API to capture navigation events and ensure
-that only pages to approved sites can be loaded within that controlled frame.
+If the "controlled-frame" feature is enabled for an IWA in its manifest, each
```

Comment:
I'd recommend referring to IsolatedContexts instead of Isolated Web Apps, which are how IsolatedContexts are exposed, but for APIs it's the IsolatedContext that matters. 

I _think_ the more correct framing is that the Controlled Frame element is exposed to any [=document=] with the "controlled-frame" [=policy-controlled feature=] whose [=environment settings object=] is an [=isolated context=].

The "with the ... feature" and "whose environment settings object" parts both might be a bit hand-wavy.

Reply:
Agreed, thanks!